boa_engine/src/vm/code_block.rs

```diff
@@ -1436,8 +1448,8 @@ impl JsObject {
                             .expect("this binding must be object")
                             .clone())
                     } else {
-                        //context.throw_type_error("Function constructor must not return non-object")
-                        context.throw_reference_error("Must call super constructor in derived class before accessing 'this' or returning from derived constructor")
+                        //Err(JsNativeError::typ().with_message("Function constructor must not return non-object").into())
```

Comment:
Why is this commented out?

Reply:
Oh, I was debugging and forgot to remove the cause of the bug 😅

Reply:
Oh, I was debugging and forgot to remove the cause of the bug 😅